projects/etos_suite_runner/src/etos_suite_runner/__main__.py

```diff
@@ -27,12 +31,39 @@
 
 def main():
     """Entry point allowing external calls."""
-    esr = ESR()
+    etos = ETOS("ETOS Suite Runner", os.getenv("SOURCE_HOST"), "ETOS Suite Runner")
+    etos.config.set("results", [])
+    esr = ESR(etos)
     try:
         esr.run()  # Blocking
+        results = etos.config.get("results") or []
+        result = None
+        for suite_result in results:
```

Comment:
Maybe make this loop easier to understand? 

Something like:

failed = any(d.get("verdict") == "FAILED" for d in dict_list)
inconclusive = not failed and any(d.get("verdict") == "INCONCLUSIVE" for d in dict_list)
successful = all(d.get("verdict") == "SUCCESSFUL" for d in dict_list)

Reply:
I added a comment to clarify why we break in the 'FAILED' case. Other than that I don't think that this suggestion is easier to understand compared to what's written.